src/lib/basic.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @fileoverview Run by all browsers in `script defer`. Used to set up
+ * Analytics, including with our known previous signed-in state that was cached
+ * in `localStorage`.
+ *
+ * This file is built with Rollup, but separately to our core bundle.
```

Comment:
Is it worth mentioning explicitly that this file should never import anything from our core bundle?

Also might be good to explicitly state it's for IE11 etc.